formant_ros2_adapter/scripts/components/subscriber/subscriber_coordinator.py

```diff
@@ -17,7 +18,7 @@ def __init__(
     ):
         self._logger = get_logger()
         self._fclient = fclient
-        self._ingester = Ingester(self._fclient)
+        self._ingester = BatchIngester(self._fclient)
```

Comment:
Lets check to see if the `_fclient` has `post_data_multi` to choose which ingester we use